### PR TITLE
 feat: add share action

### PR DESCRIPTION
**Pull request by jobvs**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -15,7 +15,7 @@ mobile.
 |                     | Open map                     |
 |                     | Open URL                     |
 |                     | Send text message            |
-|                     | Is wifi connection           |
+|                     | Share                        |
```
Comment (mmehmetAliIzci) [inline README.md]: Why did we remove the wifi connection ? Is it really gone ?

File: src/external/Share.ts [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+                return false;
+            }
+            return true;
+        });
+    }
+
+    // Web platform
+    // Documentation https://developer.mozilla.org/en-US/docs/Web/API/Navigator/share
+    if (navigator && navigator.share) {
+        return navigator.share({ url, text, title }).then(() => true);
```
Comment (mmehmetAliIzci) [inline src/external/Share.ts]: What happens if user doesnt provide any url/text/title ?

File: src/external/Share.ts [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+    }
+
+    // Hybrid platform
+    // Documentation https://github.com/EddyVerbruggen/SocialSharing-PhoneGap-Plugin
+    if (window && window.plugins && window.plugins.socialsharing) {
+        return new Promise((resolve, reject) => {
+            window!.plugins!.socialsharing!.shareWithOptions(
```
Comment (mmehmetAliIzci) [inline src/external/Share.ts]: What happens if user doesnt provide any url/text/title ?

File: README.md (hunk 1 above)
Comment (jobvs) [reply README.md]: It was never there 😅, mistakenly copy/pasted.

File: src/external/Share.ts (hunk 2 above)
Comment (jobvs) [reply src/external/Share.ts]: It will throw an error on line 24.

File: src/external/Share.ts (hunk 3 above)
Comment (jobvs) [reply src/external/Share.ts]: It will throw an error on line 24.

File: src/external/Share.ts (hunk 2 above)
Comment (diego-antonelli) [reply src/external/Share.ts]: @mmehmetAliIzci please review again

File: src/external/Share.ts (hunk 3 above)
Comment (diego-antonelli) [reply src/external/Share.ts]: @mmehmetAliIzci please review again

**Comment by shpry** [comment]
Tested

**Pull request merged**